src/Network/AWS/ARN/States.hs

```diff
@@ -0,0 +1,46 @@
+{-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module Network.AWS.ARN.States
```

Comment:
Are there any other ARNs from `states` that we should capture in this PR?

Reply:
I'm not aware of any other resources that we will use from "Step Function"